nh_eobs/report/report_nh_clinical_observation_report.py

```diff
@@ -526,6 +532,9 @@ def get_ews_observations(self, data, spell_activity_id):
     def convert_partial_reasons_to_labels(self, ews_obs):
         ews_model = self.env['nh.clinical.patient.observation.ews']
         for ews in ews_obs:
+            partial_reason = ews['values'].get('partial_reason', None)
```

Comment:
change this to `ews.get('values', {}).get('partial_reason', None)` so that even if values isn't in `ews` for whatever reason this will still work instead of causing an `AttributeError: None has no attribute .get`